.travis.yml

```diff
@@ -25,6 +25,8 @@ script: ./bin/phpunit
 addons:
   code_climate:
     repo_token: d1a6f82281cd365fc3b295539d19a932f1bd30a5d1544d6ef3074bd855351db3 
-
+  apt:
```

Comment:
to improve the readability, I would suggest to put the `addons.apt` key at the beginning of the file